circle.yml

```diff
@@ -6,7 +6,7 @@ machine:
 
 dependencies:
     pre:
-        - echo y | android update sdk --no-ui --all --filter "android-26,build-tools-26.0.0,tools,platform-tools,extra-android-m2repository"
+        - echo y | android update sdk --no-ui --all --filter "android-27,build-tools-27.0.0,tools,platform-tools,extra-android-m2repository"
```

Comment:
Build tools needs to be updated to `27.0.1` here too

Reply:
Oh that's right. Thanks for catching it.